frontend/doc-site/04__Analyze Public Data/4_4__Annotate.mdx

```diff
@@ -0,0 +1,171 @@
+
+After you successfully complete the automatic annotation workflow, you can explore and refine your predictions using [standard features in CELLxGENE](https://cellxgene.cziscience.com/docs/04__Analyze%20Public%20Data/4_1__Hosted%20Tutorials) such as:
+- visualizing clusters by predicted cell type
+- plotting marker genes
+- performing differential expression
+- refining cell type labels by creating custom annotations
+
+**Disclaimer:** The Annotation feature in CELLxGENE is new and experimental. We intend for the results generated by the workflow to be a starting point for annotating your single cell dataset. If you run into issues completing this workflow, we would love to hear about your experience. Feel free to reach out to [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) to get help running the workflow or to forward constructive feedback. In the meantime, in order to obtain the best results, we make the following recommendations:
+- use this predictive annotation workflow with data generated by 10X platforms
+- use the tissue model which is most closely related to the tissue profiled in your single cell experiment
+
+## Installation of CELLxGENE[annotate]
+
+
+You can install the latest experimental release of CELLxGENE to gain access to the automatic annotation subcommand. It is highly recommended to perform installation in a clean virtual environment (such as conda or venv) with python version 3.8+. Find examples of virtual environment setup using venv and conda below
+
+### venv setup
+
+```
+brew install pyenv  # used by MLflow
+
+virtualenv --python python3.8 /path/to/venv
+
+. /path/to/venv/bin/activate
+
+```
+
+### conda setup
+
+```
+conda create -n cellxgene 'python=3.8'
+
+conda activate cellxgene
+```
+
+### pip install
+Once your virtual environment is installed and activated, install the `annotate` feature using pip using the following command:
+
+`pip install cellxgene[annotate]`
+
+you may also install from our test PyPi release:
+
+`pip install --no-cache-dir --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple 'cellxgene[annotate]==1.1.0.dev4'`
+
+You can run the following to test the success of your installation:
+
+`cellxgene annotate -h` or `cellxgene annotate --help`
+
+## CELLxGENE Annotation Workflow
+
+See our quickstart guide to  quickly see if you can run through the pipeline or check out the more detailed sections below for more detailed descriptions of the workflow. Both assume that you have properly set up a clean virtual environment to work from.
+
+### Quickstart
+
+Depending on your environment manager, execute one of the following quickstart commands:
+
+#### venv
+```
+mkdir annotations #create a directory to download data, run annotation pipeline and store annotated object
+
+cd annotations #navigate to newly created directory
+
+wget https://github.com/chanzuckerberg/cellxgene/raw/main/example-dataset/pbmc3k.h5ad # retrieve sample dataset
```

Comment:
nit: suggest `curl` over `wget` since `curl` ships with more base packages. wget needs to be brew installed, iirc.

Reply:
will address this for the production release, but will give to pre-alpha group with wget commands

Reply:
addressed this feedback, thank you for the suggestion, resolving the thread